content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._en.md

```diff
@@ -33,21 +33,23 @@ do_list:
   role='alert'
 %}
 
-<div class="margin-top-5">
-  <span class="usa-tag usa-tag--informative">Recommended</span>
-</div>
+## Overview
 
-<div class="margin-top-2">
-  <h2 id="phone-tips" class="margin-0">
-    Use a phone to take photos of the front and back of your ID
-  </h2>
-</div>
+You must take a photo of your accepted driver’s license or state ID card and enter your personal information to verify your identity online.
+
+You may need to take a photo of yourself with a phone to verify your identity online. This is to check that you are the person on your ID. If you don’t have a phone with a camera, you can [verify your identity in person]({{ '/help/verify-your-identity/verify-your-identity-in-person/' | locale_url }}).
```

Comment:
@night-jellyfish - when I click on the link in the site preview i land at the following 🤔 
![Screenshot 2024-04-16 at 1 00 04 PM](https://github.com/GSA-TTS/identity-site/assets/1261794/80e2b406-e945-40c5-87da-b710ca5b58eb)

Reply:
I thought this might be because the test site was down, but Amir has seen this locally. This might be a problem with the way link generation is done.

Reply:
I think this is due to a conflict between two custom plugins:

- https://github.com/GSA-TTS/identity-site/blob/main/_plugins/locale_url_filter.rb
- https://github.com/GSA-TTS/identity-site/blob/main/_plugins/base_url.rb

The latter is meant to simplify use of root-relative paths in links in Markdown files. I think it's also catching the root-relative path produced by `locale_url`, which is causing the [base URL path](https://mademistakes.com/mastering-jekyll/site-url-baseurl/) to be added twice.

I think the solution is to hard-code the root-relative path in the Markdown. Since these files are locale-specific already, we don't need to use the `locale_url` helper. We'll just need to make sure we add the `/es/` and `/fr/` path prefixes to the Spanish and French versions of the Markdown file.

```suggestion
You may need to take a photo of yourself with a phone to verify your identity online. This is to check that you are the person on your ID. If you don’t have a phone with a camera, you can [verify your identity in person](/help/verify-your-identity/verify-your-identity-in-person/).
```